package.json

```diff
@@ -31,7 +31,6 @@
     "node": ">=10"
   },
   "dependencies": {
-    "@nomiclabs/hardhat-ethers": "^2.0.6",
     "@types/async-retry": "^1.4.2",
```

Comment:
The entire `@types` scope should probably also be moved to devDependencies - they are only used for compilation.